Libbna/CUSTOM-CMS
# #291- Scroll top top feature added

**Testing steps:**
1) Scroll Down on any page.
2) look if you can see the Top button on the right bottom side.
3) Click on the Top button and it will automatically scroll to top.
